db/migrate/20201107111600_create_podcast_appearances.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePodcastAppearances < ActiveRecord::Migration[6.0]
+  def change
+    create_table :podcast_appearances do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :podcast_episode, null: false, foreign_key: true
```

Comment:
this should both have `index: false` so not to create an additional index, since they are already indexed by the unique index on line 11